src/createUseContext.lua

```diff
@@ -10,11 +10,17 @@ local function createUseContext(component, useEffect, useState)
 		context.Consumer.init(fakeConsumer)
 
 		local contextEntry = fakeConsumer.contextEntry
-		local value, setValue = useState(contextEntry.value)
+		local value, setValue = useState(if contextEntry ~= nil then contextEntry.value else nil)
```

Comment:
```suggestion
		local value, setValue = useState(if contextEntry == nil then nil else contextEntry.value)
```

Just to avoid negative conditionals